daemon/algod/api/algod.oas2.json

```diff
@@ -3972,6 +4045,74 @@
           }
         }
       }
+    },
+    "SimulationUnnamedGroupResources": {
+      "description": "If unnamed resource access is allowed, this is the subset of unnamed resources that were accessed by this group and could benefit from group resource sharing. In other words, there are no restrictions on where these resources must be placed in the transaction group. Also see property 'unnamed-resources-accessed' in SimulateTransactionResult to see unnamed resources accessed by each transaction which cannot benefit from group sharing; in contrast, these resources must be placed local to that transaction.",
+      "type": "object",
+      "required": [
+        "resources"
+      ],
+      "properties": {
+        "resources": {
+          "$ref": "#/definitions/SimulationUnnamedResourceAssignment"
+        },
+        "asset-holdings": {
+          "description": "The unnamed asset holdings that were referenced. The order of this array is arbitrary.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssetHoldingReference"
+          }
+        },
+        "app-locals": {
+          "description": "The unnamed app local states that were referenced. The order of this array is arbitrary.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationLocalReference"
+          }
+        }
+      }
+    },
+    "SimulationUnnamedResourceAssignment": {
+      "description": "The set of unnamed resources that were accessed during a simulation call.",
+      "type": "object",
+      "required": [],
+      "properties": {
+        "accounts": {
+          "description": "The unnamed accounts that were referenced. The order of this array is arbitrary.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "x-algorand-format": "Address"
+          }
+        },
+        "assets": {
+          "description": "The unnamed assets that were referenced. The order of this array is arbitrary.",
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "x-algorand-format": "uint64"
+          }
+        },
+        "apps": {
+          "description": "The unnamed applications that were referenced. The order of this array is arbitrary.",
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "x-algorand-format": "uint64"
+          }
+        },
+        "boxes": {
```

Comment:
The fact that `boxes` is in here (the transaction specific type) bolsters the idea that `app-locals` and `asset-holdings` should be too. After all, boxes should never show up here for in the transaction specific result, right? Since they are always shared.